style.css

```diff
@@ -23,13 +27,68 @@ ul {
 	list-style: none;
 }
 
-a {
+a:not(.item-link) {
```

Comment:
When I did this, was for a  debug/demonstration. In the final solution, I'd remove the `a` selector completely